packages/cli/src/cli/commands/integration.ts

```
@@ -594,14 +705,326 @@ function inboundWebhookName(prefix: string): string {
+                  subscription.url === entry.url &&
+                  sameGlobSet(subscription.pathGlobs, entry.pathGlobs) &&
+                  !activeWebhookSubscriptionIds.has(subscription.subscriptionId)
+              );
+              for (const orphan of orphans) {
+                await deleteCloudSubscription(orphan.subscriptionId, entry.relayfileWorkspaceId);
+              }
+              cloudResolved = true;
+            }
+          }
+          // Relay side: recover by concrete id or deterministic key.
+          let relayResolved = Boolean(relay) && entry.relayScope === options.relayScope;
```

ESLint

[error] 'relayResolved' is never reassigned. Use 'const' instead.

(prefer-const)